pilot/pkg/model/config.go

```diff
@@ -310,6 +310,9 @@ const (
 
 	// IstioSystemNamespace is the namespace where Istio's components are deployed
 	IstioSystemNamespace = "istio-system"
+
+	// ConfigRootNamespace is a namespace for storing configurations
+	ConfigRootNamespace = "istio-config"
```

Comment:
It’s configured via helm. So why hardcode here?

Reply:
This var is not needed

Reply:
It's used in pilot/pkg/model/sidecar_test.go for a unit test. Would defining it in sidecar_test.go be better and prevent causing confusion?